pandas/_libs/groupby_helper.pxi.in

```diff
@@ -337,8 +383,13 @@ def group_rank_{{name}}(float64_t[:, :] out,
                     out[i, 0] = NAN
                 elif grp_sizes[i, 0] != 0:
                     out[i, 0] = out[i, 0] / grp_sizes[i, 0]
-{{endif}}
-{{endfor}}
+
+
+group_rank_float64 = group_rank["float64_t"]
+group_rank_float32 = group_rank["float32_t"]
+group_rank_int64 = group_rank["int64_t"]
+# Note: we do not have a group_rank_object because that would require a
```

Comment:
FWIW I don't think we really even want this #19560

Reply:
OK.  The comment seemed appropriate since we are using the same fused type.  Could add a reference to 19560 in the comment?